walletlibrary/build.gradle

```diff
@@ -21,16 +23,21 @@ android {
         minSdkVersion min_sdk_version as Integer
         targetSdkVersion target_sdk_version as Integer
         versionCode 1
-        versionName "1.0"
+        versionName _version
+        buildConfigField 'String', 'WalletLibraryVersion', "\"$_version\""
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
```

Comment:
What is minifyEnabled for?

Reply:
I will do some more testing before I turn it on. It would reduce the size of APK by removing the unused code.